tests/php/test-amp-instagram-embed.php

```diff
@@ -95,6 +103,11 @@ public function get_raw_embed_dataset() {
 				'<amp-instagram data-shortcode="BhsgU3jh6xE" layout="responsive" width="600" height="600"></amp-instagram>' . "\n\n",
 			],
 
+			'blockquote_tv_embed'                    => [
+				wpautop( '<blockquote class="instagram-media" data-instgrm-permalink="https://www.instagram.com/tv/BhsgU3jh6xE/"><div style="padding: 8px;">Lorem ipsum</div></blockquote> <script async defer src="//www.instagram.com/embed.js"></script>' ), // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript, WordPress.Arrays.ArrayDeclarationSpacing.ArrayItemNoNewLine
+				'<amp-instagram data-shortcode="BhsgU3jh6xE" layout="responsive" width="600" height="600"></amp-instagram>' . "\n\n",
```

Comment:
Aside: It would be nice if the `<blockquote>` were retained as `fallback` content inside of the `<amp-instagram>` perhaps. But this is for another day.